disnake/threads.py

```diff
@@ -182,6 +183,7 @@ def _from_data(self, data: ThreadPayload):
         self.slowmode_delay = data.get("rate_limit_per_user", 0)
         self.message_count = data.get("message_count")
         self.member_count = data.get("member_count")
+        self._newly_created: Optional[bool] = data.get("newly_created")
```

Comment:
why is this saved at all? We can dispatch a on_thread_create if its true in the received data, in which case the thread will be known to be newly created and we do not need to save it.

Reply:
Because it's technically accurate

Reply:
But we don't need it. It is used in one function, and is only True in that specific gateway event, it should not be stored.

Discord's documentation does not document it as a thread attribute, there is absolutely no reason to store it as such.